src/python/pants/backend/shell/target_types.py

```diff
@@ -83,15 +88,11 @@ def binary_path_test(self) -> BinaryPathTest | None:
         return BinaryPathTest((arg,))
 
 
-class Shunit2TestsDependencies(Dependencies):
+class Shunit2TestDependenciesField(Dependencies):
     supports_transitive_excludes = True
 
 
-class Shunit2TestsSources(ShellSources):
-    default = ("*_test.sh", "test_*.sh", "tests.sh")
-
-
-class Shunit2TestsTimeout(IntField):
+class Shunit2TestTimeoutField(IntField):
```

Comment:
If I could go back, I would have added `Field` to the end of most field names. A couple extra characters, but I think it brings clarity - it's often confusing imo what is a target's field vs. a rule's helper type, e.g. `EntryPoint` from `pants.backend.target_types`.